deepmd/utils/argcheck.py

```diff
@@ -736,270 +636,367 @@ def descrpt_dpa2_args():
     doc_add_tebd_to_repinit_out = "Add type embedding to the output representation from repinit before inputting it into repformer."
     return [
         # repinit args
-        Argument("repinit_rcut", float, doc=doc_repinit_rcut),
-        Argument("repinit_rcut_smth", float, doc=doc_repinit_rcut_smth),
-        Argument("repinit_nsel", int, doc=doc_repinit_nsel),
+        Argument("repinit", dict, dpa2_repinit_args(), doc=doc_repinit),
+        # repformer args
+        Argument("repformer", dict, dpa2_repformer_args(), doc=doc_repformer),
+        # descriptor args
+        Argument(
+            "concat_output_tebd",
+            bool,
+            optional=True,
+            default=True,
+            doc=doc_concat_output_tebd,
+        ),
+        Argument("precision", str, optional=True, default="default", doc=doc_precision),
+        Argument("smooth", bool, optional=True, default=False, doc=doc_smooth),
```

Comment:
Why is default False?

Reply:
I've tracked about this parameter. I find it was transfered from the default value (False) of `smooth_type_embedding` param in DPA1 descriptor. To have a better performance, I suggest to change the default value here to `True`. What do you think? @wanghan-iapcm @njzjz

Reply:
default `True` is definitely a better option. 
what was the setting in dpa-2 manuscript? have you set a smooth_type_embedding?

Reply:
Yes, It was `True` in all settings we used. Because this parameter only has default `True` in class `__init__` method and was never added into argcheck.py before #3758.